sfdx-source/apex-common/main/classes/fflib_SObjectUnitOfWork.cls

```diff
@@ -121,6 +121,20 @@ public virtual class fflib_SObjectUnitOfWork
 			Database.emptyRecycleBin(objList);
 		}
     }
+
+	public virtual class UserModeDML extends SimpleDML{
```

Comment:
@daveespo -- Would it not be better to set `UserModeDML` to implement `IDML` instead of extending `SimpleDML`??